spec/features/users_creates_new_account_spec.rb

```diff
@@ -1,5 +1,28 @@
 require "rails_helper"
 
+feature "admin user accesses #index" do
```

Comment:
This scenario fits in a new file: `spec/features/admin_accesses_users_list_spec.rb`

Reply:
Done in sha: b2bcac4